src/site/content/en/metrics/fcp/index.md

```diff
@@ -41,6 +41,18 @@ Paint (FCP) and _[Largest Contentful Paint (LCP)](/lcp/)_
 &mdash;which aims to measure when the page's main contents have finished
 loading.
 
+<picture>
```

Comment:
putting the image here (above `### What is a good FCP score?`) matches the other CWV articles, but makes it harder to link to _a_ scoring section even though it's all in one spot.

One option to avoid any article reworking is to just add an id to the image, though obviously that's less future-proof

Reply:
Agreed, and as I mentioned above I think let's do that in a separate PR just to keep this one simple.